### PR TITLE
Update IAM tutorial to use channel configuration

### PR DESCRIPTION
**Pull request by spoorthipujariadobe**
## Description

Update IAM tutorial to use channel configuration

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [ ] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: docs/tutorials/ajo/create-channel-configuration.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+4. Click the **Create channel configuration** button in the top right.
+
+| ![Create new channel configuration](assets/channel-new.png?raw=true) |
+| :---: |
+| **Create new channel configuration** |
+
+5. Give the channel configuration an identifying name and description (**1**). 
+6. From the **Select channel** drop down, select **In-app messaging** (**2**).
+7. Under the In-app messaging settings section, select the platform you're targeting. For iOS, check **iOS** (**3**) and enter the tutorial app's bundle identifier in the **App id** field under **iOS** (**4**). For Android, check **Android** (**5**) and enter the tutorial app's app id in the **App id** field under **Android** (**6**).
```
Comment (rymorale) [inline docs/tutorials/ajo/create-channel-configuration.md:27]: should we update this to remove iOS references as this is the android messaging repo?

File: docs/tutorials/ajo/create-iam.md [hunk 2]
```diff
@@ -4,36 +4,35 @@
...
+| **Create a new Campaign** |
 
-3. Select **Campaigns** (**1**) in the left navigation window.
+### Choose type of campaign
 
-4. Click the **Create campaign** button (**2**) on the upper right of the screen.
+1. In the **Choose type of campaign** popup, select **Scheduled - Marketing**
 
-| ![Create a new Campaign](assets/ajo-create-campaign.png?raw=true) |
+| ![Choose type pf campaign](assets/ajo-choose-campaign-type.png?raw=true) |
```
Comment (rymorale) [inline docs/tutorials/ajo/create-iam.md:19]: typo in `of` "type pf campaign"

Comment (rymorale) [reply docs/tutorials/ajo/create-iam.md:19]: at line 21 below as well

File: docs/tutorials/ajo/create-iam.md [hunk 3]
```diff
@@ -63,10 +62,13 @@
 
     - Track action event happens, and
 
-    - Action equals **50off**
+    - Action equals **50off**    
 
 11. Hit the **Done** button (**2**).
 
+> [!NOTE]
+> For this example, we will show the message to everyone, every time an SDK call of `MobileCore.trackAction("50off", nil)` is made.
```
Comment (rymorale) [inline docs/tutorials/ajo/create-iam.md:70]: nit: use `null` for android examples

**Review by rymorale — APPROVED** [review]
LGTM. just a minor typo to fix and some iOS / Android terminology to resolve

File: docs/tutorials/ajo/create-channel-configuration.md (hunk 1 above)
Comment (spoorthipujariadobe) [reply docs/tutorials/ajo/create-channel-configuration.md:27]: The numbering is in the image 😶

**Pull request merged**